SPECS/nspr/nspr.spec

```diff
@@ -16,34 +18,31 @@ for system level and libc like functions.
 %package        devel
 Summary:        Header and development files for nspr
 Requires:       %{name} = %{version}-%{release}
+
 %description    devel
 It contains the libraries and header files to create applications
 
 %prep
-%setup -q
+%autosetup
 cd nspr
 sed -ri 's#^(RELEASE_BINS =).*#\1#' pr/src/misc/Makefile.in
 sed -i 's#$(LIBRARY) ##' config/rules.mk
 
 %build
 cd nspr
-./configure \
-    --prefix=%{_prefix} \
-    --bindir=%{_bindir} \
+%configure \
     --with-mozilla \
     --with-pthreads \
-    $([ $(uname -m) = x86_64 ] && echo --enable-64bit) \
+    --enable-64bit \
```

Comment:
Does this change mean the package builds without issues on ARM64 with this switch there?